library/ssl_tls13_server.c

```diff
@@ -4454,7 +4454,7 @@ int mbedtls_ssl_handshake_server_step( mbedtls_ssl_context *ssl )
             /* ----- WRITE SERVER CERTIFICATE VERIFY ----*/
 
         case MBEDTLS_SSL_CERTIFICATE_VERIFY:
```

Comment:
Does it make sense to rename the state enum as well?

Reply:
@hanno-arm , what do you think? [MBEDTLS_SSL_CERTIFICATE_VERIFY](https://github.com/hannestschofenig/mbedtls/blob/tls13-prototype/include/mbedtls/ssl.h#L663) is also a state name used in TLS 1.2 state machine.

I can change `MBEDTLS_SSL_CERTIFICATE_VERIFY` to `MBEDTLS_SSL_WRITE_CERTIFICATE_VERIFY` for TLS 1.3 state machine, it may make things more clear.

Reply:
We still need to figure out how to consolidate the state machine enums for TLS 1.2 and TLS 1.3. Let's leave it as-is for now.